test/types/asset-d.ts

```diff
@@ -12,89 +12,47 @@ import {
   AssetFields,
   AssetFile,
   AssetWithAllLocales,
-  EntrySys,
   GenericAsset,
   GenericAssetCollection,
   ConfiguredAsset,
   ConfiguredAssetCollection,
 } from '../../lib'
 import { ChainOption } from '../../lib/utils/client-helpers'
-
-const stringValue = ''
-const numberValue = 123
-const dateValue = '2018-05-03T09:18:16.329Z'
-
-const entrySysValue: EntrySys = {
-  contentType: { sys: { id: stringValue, type: 'Link', linkType: 'ContentType' } },
-  environment: { sys: { id: stringValue, type: 'Link', linkType: 'Environment' } },
-  revision: numberValue,
-  space: { sys: { id: stringValue, type: 'Link', linkType: 'Space' } },
-  type: '', //?
-  updatedAt: dateValue,
-  id: stringValue,
-  createdAt: dateValue,
-}
-const metadataValue = { tags: [] }
+// @ts-ignore
+import * as mocks from './mocks'
```

Comment:
The error is that we’re only allowed to import from `lib` where we definitely don’t want the mocks. I’m doing a `*` import to make sure I’m not hiding other errors like mock data not being exported.